packages/complex-matcher/src/index.js

```diff
@@ -599,33 +599,39 @@ export const parse = parser.parse.bind(parser)
 
 // -------------------------------------------------------------------
 
-const _getPropertyClauseStrings = ({ child }) => {
+const _getPropertyClauseStringsOrRegex = ({ child }) => {
   if (child instanceof Or) {
     const strings = []
     child.children.forEach(child => {
       if (child instanceof StringNode) {
         strings.push(child.value)
       }
+      if (child instanceof RegExpNode) {
+        strings.push(child.re.source.replace(/^(\^)|\\|\$$/g, ''))
+      }
     })
     return strings
   }
 
   if (child instanceof StringNode) {
     return [child.value]
   }
+  if (child instanceof RegExpNode) {
```

Comment:
I think we should more conservative otherwise we will break non-trivial regexp.

Maybe you could try to un-ecape the regexp and then re-escape it to make sure it is properly handled?

Reply:
I push a new version with a test. But if you add a regex like `/.*/` and, you click on a tag, we lose the first selector and keep only the tag. As currently on master.

Reply:
Fine by me :slightly_smiling_face: